src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -867,40 +867,75 @@ length(title) < 5
 
-Page blocks have a refresh option to automatically renew their data without having to reload the entire page. 
-
-This is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
+Page blocks have a refresh option to automatically renew their data without having to reload the entire page, this is useful for page blocks that display data that changes frequently, such as a list of new leads, an update to a chart, or a list of recent comments.
 
 [IMPORTANT]
 ====
-Automation, File, Content, Nylas, Navigation, and Geometry blocks do not have the refresh option.
+Automation, file, content, nylas, navigation, and geometry blocks do not have the refresh option.
 ====
 
-
 === Enabling Auto-refresh
 
 When set, page blocks that have this configuration will automatically refresh based on a set interval.
 
-To activate auto refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab, there you will find the refresh option.
+To activate auto-refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab. There you will find the refresh option.
 
-You'll want to enter a numerical value into the input above the *Show refresh button* toggle, click save and close, save the page and you should see the block start to every time your interval elapses.
+You'll want to enter a value expressed in seconds into the input above the *Show refresh button* toggle, click save and close, save the page and you should see the block start to refresh when your interval elapses.
 
 The lowest interval allowed is 5 seconds. 
-Setting an interval lower than that would make the input default to 5 seconds. 
 To turn auto-refresh off on a block, you need to set its value to 0.
 
-image:compose-configuration/page-block-general.png["Page block general tab", role="data-zoomable"]
+.Page Block General Tab
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-block-general.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 0,
+    "w": 1920,
+    "h": 1080
+  }
+}
+----
+
 
 === Enabling Manual Refresh
 
-To activate, open a page builder, edit a page block. 
-In its general tab, find and toggle the *Show refresh button* switch to its on state. 
+To activate auto-refresh, navigate to the page builder, choose a block that supports refresh, and go to its general tab. There you will find a toggle that reads *Show refresh button*.
+Switch it to its on state. 
 
-When set, page blocks that have this configuration will have a refresh button in the top right corner of the page block, allowing the user to manually refresh the page block. 
+When set, page blocks that have this configuration will have a refresh button in the top right corner of the page block as highlighted in the image below, allowing the user to manually refresh the page block. 
```

Comment:
This is OK but what I prefer to do is I place the screenshot right after the paragraph and omit "as shown here, there, somewhere".

What I'd do here is I'd move the `To turn manual refresh off on a block...` paragraph after the screenshot and remove the `as highlighted in the image below`